internal/crowdactions/crowdaction.go

```diff
@@ -0,0 +1,81 @@
+package crowdaction
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/CollActionteam/collaction_backend/internal/constants"
+	"github.com/CollActionteam/collaction_backend/internal/models"
+	"github.com/CollActionteam/collaction_backend/utils"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
```

Comment:
We should avoid using aws dependencies outside of the aws repositories.

Reply:
This is a very good point, the thing is that these dependencies are used by the crowdaction's internal. If this is the case, should I move this piece of logic somewhere else?

For example: 
The expression package is being used for the filter condition

`
"github.com/aws/aws-sdk-go/service/dynamodb/expression"
filterCond := expression.Name(KeyDateJoinBefore).GreaterThan(expression.Value(utils.GetDateStringNow()))
`

Reply:
Yeah.. that is indeed an issue.
I'm not sure if it's worth it to create stand-in objects to be used outside of the aws repositories for everything.
Maybe we should make an exception for DynamoDB, since we will (probably) never use a different database and thus tight coupling might be more permissible. (?)
@mrsoftware what is your view of this?